src/js/Content/Features/Community/ProfileHome/FCustomBackground.js

```diff
@@ -6,18 +6,7 @@ export default class FCustomBackground extends Feature {
         const prevHash = window.location.hash.match(/#previewBackground\/(\d+)\/([a-z0-9.]+)/i);
         if (prevHash) {
             const imgUrl = `//steamcdn-a.akamaihd.net/steamcommunity/public/images/items/${prevHash[1]}/${prevHash[2]}`;
-
-            const testImg = document.createElement("img");
-            testImg.style.display = "none";
-            testImg.src = imgUrl;
-
-            document.body.append(testImg);
-
-            // Make sure the url is for a valid background image
-            testImg.addEventListener("load", () => {
-                this._setProfileBg(imgUrl);
-                testImg.remove();
-            });
```

Comment:
I couldn't get background preview to work without removing this. Will there be problems if the image can't load?

Reply:
I assume that your problem was that you didn't wait for the load event to fire and thus it would overwrite the background after you pinned it.
But I suppose it isn't necessary to do this precautious check here, since the URLs are either static or directly come from the server